src/wp-admin/includes/file.php

```diff
@@ -1676,101 +1752,127 @@ function request_filesystem_credentials( $form_post, $type = '', $error = false,
-// Print a H1 heading in the FTP credentials modal dialog, default is a H2.
-$heading_tag = 'h2';
-if ( 'plugins.php' === $pagenow || 'plugin-install.php' === $pagenow ) {
-	$heading_tag = 'h1';
-}
-echo "<$heading_tag id='request-filesystem-credentials-title'>" . __( 'Connection Information' ) . "</$heading_tag>";
-?>
-<p id="request-filesystem-credentials-desc"><?php
-	$label_user = __('Username');
-	$label_pass = __('Password');
-	_e('To perform the requested action, ClassicPress needs to access your web server.');
+	<?php
+	// Print a H1 heading in the FTP credentials modal dialog, default is a H2.
+	$heading_tag = 'h2';
+	if ( 'plugins.php' === $pagenow || 'plugin-install.php' === $pagenow ) {
+		$heading_tag = 'h1';
+	}
+	echo "<$heading_tag id='request-filesystem-credentials-title'>" . __( 'Connection Information' ) . "</$heading_tag>";
+	?>
+<p id="request-filesystem-credentials-desc">
+	<?php
+	$label_user = __( 'Username' );
+	$label_pass = __( 'Password' );
+	_e( 'To perform the requested action, ClassicPress needs to access your web server.' );
 	echo ' ';
 	if ( ( isset( $types['ftp'] ) || isset( $types['ftps'] ) ) ) {
 		if ( isset( $types['ssh'] ) ) {
-			_e('Please enter your FTP or SSH credentials to proceed.');
-			$label_user = __('FTP/SSH Username');
-			$label_pass = __('FTP/SSH Password');
+			_e( 'Please enter your FTP or SSH credentials to proceed.' );
+			$label_user = __( 'FTP/SSH Username' );
+			$label_pass = __( 'FTP/SSH Password' );
 		} else {
-			_e('Please enter your FTP credentials to proceed.');
-			$label_user = __('FTP Username');
-			$label_pass = __('FTP Password');
+			_e( 'Please enter your FTP credentials to proceed.' );
+			$label_user = __( 'FTP Username' );
+			$label_pass = __( 'FTP Password' );
 		}
 		echo ' ';
 	}
-	_e('If you do not remember your credentials, you should contact your web host.');
-?></p>
+	_e( 'If you do not remember your credentials, you should contact your web host.' );
+
+	$password_value = '';
+	if ( defined( 'FTP_PASS' ) ) {
+		$password_value = '*****';
+	}
+	?>
+</p>
 <label for="hostname">
-	<span class="field-title"><?php _e( 'Hostname' ) ?></span>
-	<input name="hostname" type="text" id="hostname" aria-describedby="request-filesystem-credentials-desc" class="code" placeholder="<?php esc_attr_e( 'example: www.wordpress.org' ) ?>" value="<?php echo esc_attr($hostname); if ( !empty($port) ) echo ":$port"; ?>"<?php disabled( defined('FTP_HOST') ); ?> />
+	<span class="field-title"><?php _e( 'Hostname' ); ?></span>
+	<input name="hostname" type="text" id="hostname" aria-describedby="request-filesystem-credentials-desc" class="code" placeholder="<?php esc_attr_e( 'example: www.wordpress.org' ); ?>" value="
+																																						<?php
+																																						echo esc_attr( $hostname );
+																																						if ( ! empty( $port ) ) {
+																																							echo ":$port";}
+																																						?>
```

Comment:
indentation
ending brace should be on new line

Reply:
Agreed, will add to PR update.